core/src/main/java/io/kestra/core/schedulers/AbstractScheduler.java

```diff
@@ -815,7 +823,7 @@ private static ZonedDateTime now() {
 
     private Optional<SchedulerExecutionWithTrigger> evaluateScheduleTrigger(FlowWithWorkerTrigger flowWithTrigger) {
         try {
-            FlowWithWorkerTrigger flowWithWorkerTrigger = flowWithTrigger.from(pluginDefaultService.injectDefaults(
+            FlowWithWorkerTrigger flowWithWorkerTrigger = flowWithTrigger.from(pluginDefaultService.injectAllDefaults(
```

Comment:
Don't we already have the defaults as they are injected inside the flow listener consumer?